### PR TITLE
DEMO-230: Add missing templates

### PR DESCRIPTION
**Pull request by ciastektk**
https://jira.ez.no/browse/DEMO-230

### CODE REVIEW

File: app/Resources/views/themes/tastefulplanet/blocks/contentlist/cards.html.twig [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{% extends ':blocks:_block.html.twig' %}
+
+{% block block_identifier %}schedule{% endblock %}
```
[the remark this answers is not in the document]
Comment (ciastektk) [reply app/Resources/views/themes/tastefulplanet/blocks/contentlist/cards.html.twig]: Block identifier is incorrect. I forget to change name of it.

File: app/Resources/views/themes/tastefulplanet/blocks/contentlist/list.html.twig [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{% extends ':blocks:_block.html.twig' %}
+
+{% block block_identifier %}schedule{% endblock %}
```
[the remark this answers is not in the document]
Comment (ciastektk) [reply app/Resources/views/themes/tastefulplanet/blocks/contentlist/list.html.twig]: As in the /contentlist/cards.html.twig case

File: app/Resources/views/themes/tastefulplanet/blocks/tastes/list.html.twig [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{% extends ':blocks:_block.html.twig' %}
+
+{% block block_identifier %}schedule{% endblock %}
```
[the remark this answers is not in the document]
Comment (ciastektk) [reply app/Resources/views/themes/tastefulplanet/blocks/tastes/list.html.twig]: As in the /tastefulplanet/blocks/contentlist/cards.html.twig case

File: app/config/default_layouts.yml [hunk 4]
```diff
@@ -107,8 +107,15 @@ ezplatform_page_fieldtype:
             views:
                 blog:
```
Comment (SylvainGuittard) [inline app/config/default_layouts.yml]: Do we still need this view?

Comment (ciastektk) [reply app/config/default_layouts.yml]: Yes because we are using this view in Places & Tastes landing page to display Blog posts

**Comment by damianz5** [comment]
Review and ready to merge, ping @SylvainGuittard final review please

**Pull request merged**